spec/acceptance/rabbitmq_exporter_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper_acceptance'
+
+describe 'prometheus rabbitmq_exporter' do
+  it 'rabbitmq_exporter works idempotently with no errors' do
+    pp = 'include prometheus::rabbitmq_exporter'
+    # Run it twice and test for idempotency
```

Comment:
do we really need to add this comment as we have the `it`?

Reply:
no, it isn't required